packages/fxa-payments-server/src/components/fields/index.tsx

```diff
@@ -214,19 +214,27 @@ export const Input = withLocalization(UnwrappedInput);
 
 type StripeElementProps = { onValidate?: OnValidateFunction } & FieldProps & {
     component: any;
+    getString: Function | undefined;
   } & ReactStripeElements.ElementProps;
 
 export const defaultStripeElementValidator: OnValidateFunction = (
   value,
   focused,
-  props
+  props,
+  getString
 ) => {
   if (!value || value.empty) {
     if (props.required) {
       return {
         value,
         valid: false,
-        error: focused ? null : `${props.label} is required`,
+        error: focused
+          ? null
+          : `${
+              getString
+                ? getString('input-error-is-required', { label: props.label })
+                : props.label + ' is required'
```

Comment:
@lmorchard updated, I still have the concatenated fallback, it seems it still gets the message across if the translation doesn't exist.